setup.py

```diff
@@ -10,7 +10,7 @@
 except ImportError:
     pass
 
-VERSION = '0.4.7'
+VERSION = '0.4.6+work4.2'
```

Comment:
'work4.2' looks REALLY weird, it looks like version '4.2'. Unavoidable? ✅

Reply:
I'm afraid so. Our "slug" is awkward for most short references. :-P